hedera-node/hedera-app/src/main/java/com/hedera/node/app/config/BootstrapConfigProviderImpl.java

```diff
@@ -60,7 +59,6 @@ public BootstrapConfigProviderImpl() {
                 .withConfigDataType(LedgerConfig.class)
                 .withConverter(new BytesConverter())
                 .withConverter(new SemanticVersionConverter())
-                .withConverter(new ProfileConverter())
```

Comment:
So these are automatically converted now  without converter explicitly as I understand. Is that correct ?